orchestrator-salt/src/main/resources/salt/salt/log4jpatch/init.sls

```diff
@@ -0,0 +1,10 @@
+{% if salt['pillar.get']('log4j_patch_enabled',False) == True %}
+run_log4j_patch:
+  cmd.script:
+    - name: patch
+    - source: salt://log4jpatch/scripts/cm_cdp_cdh_log4j_jndi_removal.sh
```

Comment:
do we log the results somewhere?

Reply:
I think there is logs in minion log, but the scripts does not logging into file unfortunately

Reply:
you could use `tee` to have the output in a file and in minion log, for easier debugging